canonical-web-and-design/snapcraft.io
# Release UI - refactor in preparation for drag n drop

Fixes #1960 

Splitting release UI into smaller components to allow future drag and drop implementation to wrap around them.
Also moving some of the release UI state to redux.

### QA

- ./run or  https://snapcraft-io-canonical-web-and-design-pr-1988.run.demo.haus/
- go to Releases page of any snap
- check if everything works as before
- make sure you can click on any cell to open history panel
- try selecting some available revisions
- try promoting some channel
- try switching to different track...
